lightning/log/redact.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
```

Comment:
shall we directly shall the package in br? Since they should be almost the same

Reply:
Some functions are not implemented in br's package. Since these two repositories are going to be merged together, I think we can do the refactor at that time to avoid update the dependencies now.

Reply:
@overvenus FYI about this task.